libc/src/sys/stat/linux/kernel_statx.h

```diff
@@ -73,7 +73,7 @@ LIBC_INLINE int statx(int dirfd, const char *__restrict path, int flags,
                       struct stat *__restrict statbuf) {
   // We make a statx syscall and copy out the result into the |statbuf|.
   ::statx_buf xbuf;
-  int ret = LIBC_NAMESPACE::syscall_impl<int>(SYS_statx, dirfd, path, flags,
+  int ret = LIBC_NAMESPACE::syscall_impl<int>(0, dirfd, path, flags,
```

Comment:
What's the motivation for this change?

Reply:
This was leftover scratch work when I was trying to build libc using the toolchain cache file. I have a different cmake invocation though that allows me to run these tests on linux.